apps/src/templates/projects/project-card.module.scss

```
@@ -34,12 +38,38 @@
   color: $neutral-dark;
 }
 
-.image {
-  flex-shrink: 0;
-  width: 100%;
-  height: 100%;
+.bold {
+  @include main-font-semi-bold;
 }
 
-.bold {
+.title {
   @include main-font-semi-bold;
-}
\ No newline at end of file
+  padding-left: 15px;
+  padding-right: 10px;
+  padding-top: 18px;
+  padding-bottom: 5px;
```

Nit: this can be consolidated
```suggestion
  padding: 18px 10px 5px 15px;
```